MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -5056,50 +5027,10 @@ private void fgetc()
         /// </summary>
         private void stricmp()
         {
+            var string1 = GetParameterString(0);
+            var string2 = GetParameterString(2);
 
-            var string1Pointer = GetParameterPointer(0);
-            var string2Pointer = GetParameterPointer(2);
-
-            var string1 = Module.Memory.GetString(string1Pointer, true);
-            var string2 = Module.Memory.GetString(string2Pointer, true);
-
-#if DEBUG
-            _logger.Info(
-                $"Comparing ({string1Pointer}){Encoding.ASCII.GetString(string1)} to ({string2Pointer}){Encoding.ASCII.GetString(string2)}");
-#endif
-
-            if (string1.Length == 0)
-            {
-                Registers.AX = 0xFFFF;
-                return;
-            }
-
-            if (string2.Length == 0)
-            {
-                Registers.AX = 1;
-                return;
-            }
-
-            for (var i = 0; i < string1.Length; i++)
-            {
-                //We're at the end of string 2, string 1 is longer
-                if (i == string2.Length)
-                {
-                    Registers.AX = 1;
-                    return;
-                }
-
-                if (string1[i] == string2[i]) continue;
-                if (string1[i] == string2[i] + 32) continue;
-                if (string1[i] == string2[i] - 32) continue;
-
-                //1 < 2 == -1 (0xFFFF)
-                //1 > 2 == 1 (0x0001)
-                Registers.AX = (ushort)(string1[i] < string2[i] ? 0xFFFF : 1);
-                return;
-            }
-
-            Registers.AX = 0;
+            Registers.AX = (ushort)(short)String.Compare(string1, string2, /* ignoreCase= */ true);
```

Comment:
`string` vs `String` alias
`(short)` casting can be removed
You can use parameter naming in C#, so `ignoreCase: true` would work for the 3rd parameter vs. the inline comment

Reply:
Cool, using magic numbers like true/false in method calls can be confusing if you're not sure what the parameter is for, but parameter naming takes that ambiguity away